Kudu.SiteManagement/SiteManager.cs

```diff
@@ -171,7 +171,7 @@ private static List<string> GetSiteUrls(IIS.Site site)
                 {
                     try
                     {
-                        DeleteSiteAsync(applicationName).Wait();
+                        await DeleteSiteAsync(applicationName);
```

Comment:
Wasn't sure if it should go in as a separate PR? It got it's own commit though. Shall I reverse it?